src/pages/test-report/components/manoeuvres-popover/manoeuvres-popover.scss

```diff
@@ -1,9 +1,19 @@
 .popover {
-  background-color: map-get($colors, 'mes-white');
-  width: 100vw;
-  height: 240px;  
-  left: 0px;
+  ion-grid {
+    border: 0px none #fdfeff !important;
```

Comment:
Could you add a class to this selector to improve specificity and eliminate `!important`?

Reply:
corrected